client/src/main/App.tsx

```diff
@@ -306,6 +306,7 @@ class App extends React.Component<AppProps, AppState> {
             updateSettings={updateSettings}
             appTheme={appTheme}
             setTheme={setTheme}
+            loggedIn={!!user}
```

Comment:
That is a clever use of the not operator. lol

Reply:
wasn't my idea lol, eslint suggested this instead of `user ? true : false`

Reply:
Eslint doing a good job here then haha nice